redis/src/cluster_async/mod.rs

```diff
@@ -586,21 +586,20 @@ impl<C> Future for Request<C> {
                         }
                         .into()
                     }
-                    ErrorKind::BusyLoadingError => Next::RetryBusyLoadingError {
+                    crate::types::RetryMethod::WaitAndRetryOnPrimaryReconnectOnSecondary => {
```

Comment:
secondary == replica? 
We're not reconnecting if it's a replica, we're redirecting to a differet replica

Reply:
maybe WaitAndRetryOnPrimaryRedirectOnReplica